sdk/core/azure-core/azure/core/tracing/__init__.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-from ._abstract_span import AbstractSpan, SpanKind, HttpSpanMixin
+from azure.core.tracing._abstract_span import AbstractSpan, SpanKind, HttpSpanMixin
 
```

Comment:
Why make this change?

Reply:
Technically, this is a different package (this is imported from azure-core), so that should be absolute import. It works only because in most of the scenarios the wheels are installed together in the same folder. We could imagine a setuptools sdist install that would broken because of that (but it's rare nowadays)